data.js

```diff
@@ -1365,4 +1365,16 @@ module.exports = [
       'Technical',
     ]
   },
+  {
```

Comment:
Could you please add your blog in a random position in the file, just to avoid merging errors.
In the middle of the file for example is okey :) Thanks a lot in advance!

Reply:
Sure, Thanks!